guides/common/modules/proc_registering-a-host-using-the-bootstrap-script.adoc

```diff
@@ -4,7 +4,7 @@
 **Deprecated** Use registration via Global registration feature.
 
 Use the bootstrap script to automate content registration and Puppet configuration.
-You can use the bootstrap script to register new hosts, or to migrate existing hosts to {ProjectNameX} from {Project} 5, RHN, SAM, or RHSM.
+You can use the bootstrap script to register new hosts, or to migrate existing hosts to {ProjectName} 6 from {ProjectName} 5, RHN, SAM, or RHSM.
```

Comment:
I wonder about `{ProjectName} 6` here. That will be odd once Satellite 7 is out. However, a bare `{ProjectName}` will also read weird. Perhaps just leave out the `to {ProjectName} 6` bit altogether?
```suggestion
You can use the bootstrap script to register new hosts, or to migrate existing hosts from {ProjectName} 5, RHN, SAM, or RHSM.
```

Reply:
Will there be an option to migrate hosts from Sat 5 to Sat 7 directly?

Reply:
I don't see why not. Satellite 6 and 7 are the same technology stack. 6.9 and 6.10 differ more than 6.10 and 7.0.

Reply:
In that case, we will need to look deeper into the phrasing once we get to updates for 7.0. For now, I would keep only this change, if you agree.

Reply:
Hang on. I was asking about version five, not six.

Reply:
I get that, but I was saying that 6 and 7 are so much the same that the bootstrap script will work with both.

Reply:
Oki. I'm removing the "to". And I'm thinking we should probably keep the version number in this case. Agreed?